src/main/java/com/spotify/google/cloud/pubsub/client/Publisher.java

```diff
@@ -118,7 +135,10 @@ private Publisher(final Builder builder) {
     this.concurrency = builder.concurrency;
     this.batchSize = builder.batchSize;
     this.queueSize = Optional.ofNullable(builder.queueSize).orElseGet(() -> batchSize * 10);
+    this.maxLatencyMs = builder.maxLatencyMs;
     this.listener = builder.listener == null ? new ListenerAdapter() : builder.listener;
+    this.scheduler = Optional.ofNullable(builder.scheduler).orElseGet(
```

Comment:
Clunky. I wish java would introduce something like the kotlin elvis operator: `x :? y`. Turns out scheduled executor injection wasn't needed anyway so removed this.